components/pickerbutton/stories/pickerbutton.stories.js

```diff
@@ -16,10 +16,91 @@ export default {
       options: ["s", "m", "l", "xl"],
       control: "select"
     },
+    label: {
+      name: "Label",
+      type: { name: "string" },
+      table: {
+        type: { summary: "string" },
+        category: "Component",
+      },
+      control: { type: "text" },
+    },
+    isOpen: {
+      name: "Open",
+      type: { name: "boolean" },
+      table: {
+        type: { summary: "boolean" },
+        category: "Component",
+      },
+      control: "boolean",
+    },
+    isRounded: {
+      name: "Rounded",
+      type: { name: "boolean" },
+      table: {
+        type: { summary: "boolean" },
+        category: "Component",
+      },
+      control: "boolean",
+    },
+    isWorkflowIconOnly: {
+      name: "Workflow icon only",
+      type: { name: "boolean" },
+      table: {
+        type: { summary: "boolean" },
+        category: "Component",
+      },
+      control: "boolean",
+    },
+    isUIIconOnly: {
+      name: "UI icon only",
+      type: { name: "boolean" },
+      table: {
+        type: { summary: "boolean" },
+        category: "Component",
+      },
+      control: "boolean",
+    },
+    isInvalid: {
+      name: "Invalid styling",
+      type: { name: "boolean" },
+      table: {
+        type: { summary: "boolean" },
+        category: "Component",
+      },
+      control: "boolean",
+    },
+    isQuiet: {
+      name: "Quiet styling",
+      type: { name: "boolean" },
+      table: {
+        type: { summary: "boolean" },
+        category: "Component",
+      },
+      control: "boolean",
+    },
+    isDisabled: {
+      name: "Disabled",
+      type: { name: "boolean" },
+      table: {
+        type: { summary: "boolean" },
+        category: "State",
+      },
+      control: "boolean",
+    },
   },
   args: {
-    rootClass: "spectrum-",
+    rootClass: "spectrum-PickerButton",
+    customClasses: ["spectrum-PickerButton--right"],
```

Comment:
```suggestion
    customClasses: ["spectrum-PickerButton--right"],
```

Oh interesting use-case here. I'd typically used customClasses solely for passing down contextual classes from parent elements but this is an interesting idea.  Would you ever want to offer the direction as a control above?

Reply:
Oh, you know: we don't have an example of `--left` on the current docs site, but there's CSS written for it, and it's something that SWC has on their docs. I think this should probably be a knob, and the class can toggle between the two directions, with `--right` being the default. I'll make that change.